### PR TITLE
refactor(lint): fix linting errors

### PR DESCRIPTION
**Pull request by chethanbellechalu-intel**

### CODE REVIEW

File: src/core/RFBStateProcessors/FrameBufferBellServerCutText.ts [hunk 1]
```diff
@@ -13,7 +13,7 @@ class FrameBufferBellServerCutText implements IStateProcessor {
   wsSocket: ICommunicator
   next: IStateProcessor
   cmdSize: number
-  binaryEncDec: TypeConverter
+  // binaryEncDec: TypeConverter
```
Comment (rsdmike) [inline src/core/RFBStateProcessors/FrameBufferBellServerCutText.ts:16]: is this no longer needed?

File: src/core/Utilities/CommsHelper.ts [hunk 2]
```diff
@@ -8,8 +8,8 @@ import { Desktop } from '../Desktop'
 import { TypeConverter } from '../Converter'
 import { ICommunicator } from '../ICommunicator'
 import { isTruthy } from './UtilityMethods'
-export class CommsHelper {
-  static sendRefresh (parent: Desktop, comm: ICommunicator): void {
+export const CommsHelper = {
```
Comment (rsdmike) [inline src/core/Utilities/CommsHelper.ts:11]: 👍

File: src/core/Utilities/ImageHelper.ts [hunk 3]
```diff
@@ -9,29 +9,29 @@ import { Desktop } from '../Desktop'
...
-export class ImageHelper {
+export const ImageHelper = {
   /**
    * puts image on canvas using the parent canvas ctx.
    * @param parent parent desktop with CTX for canvas
    * @param x x loc
    * @param y y loc
    */
-  static putImage (parent: Desktop, x: number, y: number): any {
+  putImage (parent: Desktop, x: number, y: number): any {
```
Comment (rsdmike) [inline src/core/Utilities/ImageHelper.ts:19]: shouldn't this have a return  type of void? not any?

File: src/core/Utilities/ImageHelper.ts [hunk 4]
```diff
@@ -9,29 +9,29 @@ import { Desktop } from '../Desktop'
...
+  },
 
   /**
    *
    * @param parent parent desktop
    * @param value pixel value at ptr
    * @param ptr ptr into the image pixel data
    */
-  static setPixel (parent: Desktop, value: any, ptr: number): any {
+  setPixel (parent: Desktop, value: any, ptr: number): any {
```
Comment (rsdmike) [inline src/core/Utilities/ImageHelper.ts:34]: is value a number? and return type should be void.

**Review by rsdmike — CHANGES_REQUESTED** [review]

File: src/core/RFBStateProcessors/FrameBufferBellServerCutText.ts (hunk 1 above)
Comment (chethanbellechalu-intel) [reply src/core/RFBStateProcessors/FrameBufferBellServerCutText.ts]: This variable is not being used anywhere. I have removed the commented out code

File: src/core/Utilities/ImageHelper.ts (hunk 4 above)
Comment (chethanbellechalu-intel) [reply src/core/Utilities/ImageHelper.ts]: updated

**Review by rsdmike — APPROVED** [review]

**Pull request merged**